guake/terminal.py

```diff
@@ -121,6 +121,8 @@ def __init__(self, guake):
 
         self.setup_drag_and_drop()
 
+        # GDK_BACKEND is ingnored
```

Comment:
These comment can be drop of, as `GDK_BACKEND` won't propagate to terminals.

Reply:
Edited.
Sorry I used `git push --force-with-lease`. I made a rebase to my local repo, but I don't know how to make this PR sync with my local branch for the local commit is older than the remote after rebase.

Reply:
It is fine in Guake for using `git push --force-with-lease` with rebase when doing PR.

Reply:
I didn't comment on it correctly, all of the additional comments can be removed, including the next line.

Reply:
Sorry for that. I thought it's helpful. Removed.

Reply:
Your comment on guake/main.py has been clearly stated.